frontend/src/app/sessions/service/user-session.service.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, map } from 'rxjs';
+import { Session } from 'src/app/schemes';
+import { UserService } from '../../services/user/user.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UserSessionService {
+  constructor(private userService: UserService) {}
+
+  private _sessions = new BehaviorSubject<Session[] | undefined>(undefined);
+
+  readonly sessions = this._sessions.asObservable();
+  readonly readonlySessions = this.sessions.pipe(
+    map((sessions) =>
+      sessions?.filter((session) => session.type === 'readonly')
+    )
+  );
+  readonly persistentSessions = this.sessions.pipe(
+    map((sessions) =>
+      sessions?.filter((session) => session.type === 'persistent')
+    )
+  );
```

Comment:
I don't think this is necessary. I just added the persistent session observable as it moves the filtering out of the component and provides a convenient way to get only the persistent sessions. The readonly session observable is not used at the moment, but I just added it as it might be useful in the future. Since a user usually doesn't have that many sessions, I preferred the option to just do it here in the service. Or where would you move it?